scripts/push_images.sh

```diff
@@ -2,30 +2,19 @@
 
 set -e
 
+latest_version=$(curl https://rubygems.org/api/v1/versions/decidim/latest.json | grep -o "[0-9][0-9]*.[0-9][0-9]*.[0-9][0-9]*")
```

Comment:
This might be confusing. Does automatically using the latest version satisfy a use case?

Reply:
I copied this from the `build` script. The usage in that case was being able to run the script locally without having to set any environment variables. Initially I was setting the version and fallback in the script itself, but you said you preferred to have the version specified in the CI config. So we resorted to this trick to satisfy both "cases" (version in CI config, script runnable directly).

In the case of pushing, we probably don't need it, so I can remove it  :+1: